### DISCUSSION TITLE
Looking for a team to build a Shopify Autonomous Employee

### DISCUSSION THREAD

**Discussion by KenjiPcx** [daveshap/ACE_Framework#35] [General]
Hi all, I'm Kenji. I am a final year undergraduate student, but really active in the tech space and have a decent amount of enterprise work and personal projects. Here is my linkedin https://www.linkedin.com/in/kenjiphang/

One of my ideas for a while now is to build out a bunch of shopify autonomous employees to maintain a store, from
- doing market research
- to selecting products
- to handling shipping
- to handling customer support

The influencers always say that "Dropshipping is easy" and all that, and I want to prove / disprove that 😆
I am looking for a team who would like to explore this with me, reach out if you'd like to explore, but also include how you can contribute 😄

**Comment by ippolite** [comment]
Hey Kenji! I'm Alex. You can find a bit more about me in my own call for a team - [local LLMs for some ACE layers](https://github.com/daveshap/ACE_Framework/discussions/10)

I have a question about your employees. Do you want them to be a single entity (so each employee corresponds to a layer) or to interact as a bunch of separate autonomous agents?

**Reply by KenjiPcx** [reply]
Hey Alex, nice to meet you!
I have not done the necessary research to determine which is better but based on my intuition, I would start with building just one agent (an employee contains all layers within it), and implement it as simple as possible to get a working mvp, then do exploratory research from there.
